src/appshell/qml/MainToolBar.qml

```diff
@@ -96,6 +98,7 @@ Item {
 
             checked: modelData["uri"] === root.currentUri
             title: modelData["title"]
+            isBoldOnly: (modelData["title"] === qsTrc("appshell", "Score")) && !checked && isNotationOpened
```

Comment:
I believe this is not the right way of solving the problem. Qml is not the best place for such logic

Reply:
You are right and I have seen how you dealt with it.